fastlane/Fastfile

```diff
@@ -646,4 +649,33 @@ end
 
   def simulator_version()
     return '13.3'
+  end
+
+########################################################################
+# Test Lanes
+########################################################################
+  #####################################################################################
+  # test_without_building
+  # -----------------------------------------------------------------------------------
+  # This lane runs tests without building the app.
+  # It requires a prebuilt xctestrun file and simulator destination where the tests will be run.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane test_without_building [xctestrun:<Path to xctestrun file>] [destination:<Simulator>] [try_count:<Number of times to try tests>]
+  #
+  # Example:
+  # bundle exec fastlane test_without_building xctestrun:WooCommerce_UITests_iphonesimulator13.2-x86_64.xctestrun destination:"platform=iOS Simulator,id=$SIMULATOR_UDID" try_count:3
+  #####################################################################################
+  desc "Run tests without building"
+  lane :test_without_building do | options |
+    multi_scan(
+      workspace: "WooCommerce.xcworkspace",
+      scheme: "WooCommerce",
```

Comment:
That's a good idea. The scheme varies across the lanes, so I don't think a constant makes as much sense for it, but we could define a constant for the workspace. I'm a little wary of making that change in this PR because there are variations in how the workspace is defined for different lanes, and I think it needs some testing to make sure nothing breaks. Maybe in a followup PR?

Reply:
Yeah totally 👍